lib/mutant/mutator/node/send.rb

```diff
@@ -27,7 +27,8 @@ class Send < self
           to_a:          %i[to_ary],
           to_h:          %i[to_hash],
           at:            %i[fetch],
```

Comment:
I realize this is probably outside of the scope, but should this also include `%i[fetch key?]` to be consistent with below?